templates/legacy/decisioncentral-app-secret.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 metadata:
   annotations:
     description: Examples that can be installed into your project to allow you to
-      test the Red Hat Decision Manager Central templates. You should replace the contents
+      test the Red Hat Decision Central templates. You should replace the contents
```

Comment:
Shouldn't we entirely remove this secret since it is basically the same the new one?

Reply:
Yes; removing the decisioncentral and kiserver app-secret files. The user will have to use the example-app-secret-template to generate their secrets.